### PR TITLE
Desktop_project

### PR DESCRIPTION
**Pull request by shahnawaza75**
Created Html css files
Added linters
Added images

### CODE REVIEW

**Comment by shahnawaza75** [comment]
Hi sir and mam , when i am opening project on my broswer layout is coming right fine to contact page however when i am doing it on the github deloyment link the contact us form page not looking the way it look like on my vs code project. I am attaching the vs code project layout screenshoot.
Screenshot 2022-04-02 at 7 50 56 PM
.

File: index.html [hunk 1]
```diff
@@ -132,12 +152,127 @@ <h3 class="project_name">
 </ul>
 <button class="btn1" type="submit">See Project</button>
 </div>
+</div>
 </section>
-  
+</main> -->
+<main id="work_section" class="mobile-nav">
```
Comment (OLIPLICHE) [inline index.html:158]: - [ ] You did a great job 👍 But need to fix some padding here to make your work looks like the one in Figma. 
- Please, reduce the **padding-right / padding-left** in .desk-1, .desk-2, .desk-3, .desk-4
- Please, You can also try to reduce the **margin** in .btn1 
- Kindly check if you can reduce or delete the **width** in .side_content

r2

File: index.html [hunk 2]
```diff
@@ -183,24 +330,19 @@ <h4>Frameworks</h4>
         <h4>Skills</h4>
         <img src="./images/Union.png" alt="icon">
     </div>
+    <hr>
+</div>
 
 </section>
+<section id="contactlast_section">
```
Comment (OLIPLICHE) [inline index.html:337]: - [ ] Nice job 👍 , Please try to review the **width** in **.last_para**. you can make it between (78% and 100%). It is a suggestion
- [ ] Please Reduce the **margin-left**  to around 34px in your **.last_para**. I am suggesting. 💯 
r3

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @shahnawaza75, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter result is green ✔️
- Commit messages are meaningful✔️
- The pull request have a descriptive title telling you what changes are inside ✔️
- The pull request have a short summary describing introduced changes in more detail ✔️
- node_modules folder is in the .gitignore ✔️

## Required Changes ♻️
**I made suggestions in my comments, you can also consider them.** 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please in your free time, try to fix the mobile version. you need to add the **margin (top and down)**

r1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OLIPLICHE** [comment]
> Hi sir and mam , when i am opening project on my broswer layout is coming right fine to contact page however when i am doing it on the github deloyment link the contact us form page not looking the way it look like on my vs code project. I am attaching the vs code project layout screenshoot. Screenshot 2022-04-02 at 7 50 56 PM .

Please don't make changes or work to the code while it is in the review process.

**Comment by shahnawaza75** [comment]
Unable to see the colour white after deploying the project however in my local browser colour is showing white of the contact form button
Screenshot 2022-04-03 at 12 00 29 AM
.

**Review by bimbolabuari — APPROVED** [review]
Hi @shahnawaza75,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**